app/frontend/components/pages/sendAda/sendAdaPage.tsx

```diff
@@ -45,23 +46,21 @@ const SendValidation = ({sendFormValidationError, txSuccessTab}) =>
     )
   )
 
+type Searchable = {
+  searchable: {
+    ticker: string
+    assetNameReadable: string
+  }
+}
+
 type DropdownAssetItem = Token & {
   fingerprint: string
-  assetNameHex: string
   type: AssetFamily
-  star?: boolean
-}
+} & Searchable
```

Comment:
why not just include these searchable keys besides fingerprint and type?

Reply:
It's already possible to search by fingerprint:
https://github.com/vacuumlabs/adalite/blob/add-token-registry-support/app/frontend/components/pages/sendAda/sendAdaPage.tsx#L214
I guess the Searchable type was confusing, as it was possible to search by other members as well. Initial idea was to introduce `searchable` object to hold additional information that wasn't there before and also they have no other purpose then search.